### PR TITLE
GEODE-8113: getMap never returns null

### PR DESCRIPTION
**Pull request by prettyClouds**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-redis/src/main/java/org/apache/geode/redis/internal/executor/hash/HLenExecutor.java [hunk 1]
```diff
@@ -46,12 +46,6 @@ public void executeCommand(Command command, ExecutionHandlerContext context) {
 
     try (AutoCloseableLock regionLock = withRegionLock(context, key)) {
       RedisHash map = getMap(context, key);
-
-      if (map == null) {
-        command.setResponse(Coder.getIntegerResponse(context.getByteBufAllocator(), NOT_EXISTS));
```
Comment (dschneider-pivotal) [inline geode-redis/src/main/java/org/apache/geode/redis/internal/executor/hash/HLenExecutor.java:51]: can NOT_EXISTS be removed?

File: geode-redis/src/main/java/org/apache/geode/redis/internal/executor/hash/HExistsExecutor.java [hunk 2]
```diff
@@ -55,11 +55,6 @@ public void executeCommand(Command command, ExecutionHandlerContext context) {
     ByteArrayWrapper key = command.getKey();
     try (AutoCloseableLock regionLock = withRegionLock(context, key)) {
       RedisHash map = getMap(context, key);
-
-      if (map == null || map.isEmpty()) {
-        command.setResponse(Coder.getIntegerResponse(context.getByteBufAllocator(), NOT_EXISTS));
```
Comment (dschneider-pivotal) [inline geode-redis/src/main/java/org/apache/geode/redis/internal/executor/hash/HExistsExecutor.java:60]: can NOT_EXISTS be removed?

File: geode-redis/src/main/java/org/apache/geode/redis/internal/executor/hash/HLenExecutor.java (hunk 1 above)
Comment (prettyClouds) [reply geode-redis/src/main/java/org/apache/geode/redis/internal/executor/hash/HLenExecutor.java:51]: removed `NOT_EXISTS`

File: geode-redis/src/main/java/org/apache/geode/redis/internal/executor/hash/HExistsExecutor.java (hunk 2 above)
Comment (prettyClouds) [reply geode-redis/src/main/java/org/apache/geode/redis/internal/executor/hash/HExistsExecutor.java:60]: `NOT_EXISTS` is still used a few lines down, so I left it.

**Pull request merged**